File: tmp_list.js
const fs = require('fs');

const clubsData = JSON.parse(fs.readFileSync('./data_clubs.json', 'utf8'));
const fightersData = JSON.parse(fs.readFileSync('./data_fighters.json', 'utf8'));

let output = '# Starting Clubs and Their Fighters\n\n';

for (const club of clubsData) {
    output += `## ${club.name} (Advantage: ${club.home_advantage}, Style: ${club.ai_persona})\n`;
    for (const fighterId of club.fighter_ids) {
        const fighter = fightersData.find(f => f.id === fighterId);
        if (fighter) {
            output += `- **${fighter.name}** (Age: ${fighter.age}, Archetype: ${fighter.personality.archetype})\n`;
        }
    }
    output += '\n';
}

console.log(output);

File: tmp_list.py
import json

with open("data_clubs.json", "rt", encoding="utf-8") as f:
    clubs = json.load(f)

with open("data_fighters.json", "rt", encoding="utf-8") as f:
    fighters = json.load(f)

fighter_dict = { f["id"]: f for f in fighters }

for c in clubs:
    print(f"## {c['name']} (Advantage: {c['home_advantage']}, Persona: {c['ai_persona']})")
    for fid in c["fighter_ids"]:
        if fid in fighter_dict:
            f = fighter_dict[fid]
            print(f"- **{f['name']}** - Age: {f['age']}, Archetype: {f['personality']['archetype']}")
    print("")
